### ISSUE TITLE
[Feature request]: Use an array for miniBreakIdeas

### ISSUE THREAD

**Issue by mrexodia** [hovancik/stretchly#1284]
### Version

- [ ] I'm using version 1.13.1

### Existing requests

- [X] I've checked Existing requests

### Advanced Preferences

- [X] I've checked Advanced Preferences

### What operating system are you using?

Other (specify below)

### Operating System Version

N/A

### Problem description

You are using a sequential key for the break ideas. https://github.com/hovancik/stretchly/blob/trunk/app/locales/en.json#L211

### Possible solution

Switch to using an array so you don't have to come up with an ID.

### Alternative solutions

N/A

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by hovancik** [comment]
Yeah, I would like that. I don't remember exactly, but at time either Weblate did not support arrays or I was not able to confirm that. https://hosted.weblate.org/projects/stretchly/stretchly/#repository
